WordPress/Classes/ViewRelated/Jetpack/Social/JetpackSocialNoConnectionView.swift

```diff
@@ -59,20 +59,81 @@ extension JetpackSocialNoConnectionView {
 
 // MARK: - View model
 
-struct JetpackSocialNoConnectionViewModel {
+class JetpackSocialNoConnectionViewModel: ObservableObject {
     let padding: EdgeInsets
     let hideNotNow: Bool
     let onConnectTap: (() -> Void)?
     let onNotNowTap: (() -> Void)?
+    @MainActor @Published var icons: [UIImage] = [UIImage()]
 
-    init(padding: EdgeInsets = Constants.defaultPadding,
+    init(services: [PublicizeService] = [],
+         padding: EdgeInsets = Constants.defaultPadding,
          hideNotNow: Bool = false,
          onConnectTap: (() -> Void)? = nil,
          onNotNowTap: (() -> Void)? = nil) {
         self.padding = padding
         self.hideNotNow = hideNotNow
         self.onConnectTap = onConnectTap
         self.onNotNowTap = onNotNowTap
+        updateIcons(services)
+    }
+
+    enum JetpackSocialService: String {
```

Comment:
We'll need to add Mastodon and Instagram here (along with the icons from Figma), but I'll let you decide whether to add them in this PR or do it separately.

Reply:
Yep, I can add those here! I'm going to wait for this thread to resolve though before making those changes: p1687816953471379-slack-C05362VSXFU

Reply:
I've added the social icons for Instagram and Mastodon in: https://github.com/wordpress-mobile/WordPress-iOS/pull/20823/commits/6be84eb9437c4422caf6588d35c95980407a3ed0

Additionally, I've updated Tumblr's dark icon with https://github.com/wordpress-mobile/WordPress-iOS/pull/20823/commits/007df21d17df580e178043ec3e9eef505d1ef1be after this conversation: p1687819305411249-slack-C05362VSXFU

Here are a couple screenshots (Note: I replaced Facebook's icon with Instagram here for testing):

| Light | Dark |
|------|------|
| ![Screenshot 2023-06-27 at 2 22 40 PM](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/c120c624-0056-49a0-9cb9-dc85dd5cd3c5) | ![Screenshot 2023-06-27 at 2 22 50 PM](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/0b7ed123-defe-41a7-a55f-55e061f0df3a) |